content/articles/oop-in-java-abstraction-and-encapsulation/index.md

```diff
@@ -188,29 +175,29 @@ To create a new dog object.
  
 public class Dog_Main {
  
-public static void main(String[] args) {
- 
+    public static void main(String[] args) {
```

Comment:
The code inside the main method should be on another indentation level, i.e., each line of code in the main method should have an indent before it.

Reply:
Done